pkg/config/config_darwin.go

```diff
@@ -15,3 +15,15 @@ func customConfigFile() (string, error) {
 func ifRootlessConfigPath() (string, error) {
 	return rootlessConfigPath()
 }
+
+var defaultHelperBinariesDir = []string{
+	// Homebrew install paths
+	"/usr/local/opt/podman/libexec",
```

Comment:
@Luap99 I'm afraid I wasn't clear in my last comment.  `/usr/local/bin` should also be a Homebrew install path.  I think it got deleted in the last force-push.